src/test/consensus/Consensus_test.cpp

```diff
@@ -93,6 +94,7 @@ class Consensus_test : public beast::unit_test::suite
         using namespace csf;
         using namespace std::chrono;
 
+        testcase("slow peer");
```

Comment:
Well if you didn't put a test case on every little thing, you wouldn't have to write code to squelch all the superfluous output during tests ^_^